docs/docsite/rst/playbook_guide/playbooks_tags.rst

```diff
@@ -81,6 +85,14 @@ You can apply the same tag to more than one individual task. This example tags s
 If you ran these four tasks in a playbook with ``--tags ntp``, Ansible would run the three tasks tagged ``ntp`` and skip the one task that does not have that tag.
 
 
+.. _tags_on_handlers:
+
+Adding tags to handlers
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Handlers are a special case of tasks that only execute when notified, as such they ignore all tags and cannot be selected for nor against.
```

Comment:
This could also be potentially linked from the `Limitations` section on the `Handlers` page: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_handlers.html#limitations.

Reply:
done